ui/src/app/shared/header/header.component.css

```diff
@@ -20,3 +20,9 @@
   width: 7rem;
   margin-right: .5rem;
 }
+
```

Comment:
Is it simple to fix the height of the table and give it an inline scroll? I recall seeing examples of material tables doing that. If it's not a simple style thing, fine to leave this as a TODO for another ticket.

Reply:
Looked into this a bit, but it didn't look like a trivial change so I'm going to punt for now as the PR is already large/complex.